frontend/ui/components/Space/NearProtocol/Sidebar/Sidebar.jsx

```diff
@@ -1,12 +1,18 @@
 import { Navigation } from './Navigation/Navigation.jsx';
 import cn from './Sidebar.module.scss';
 // import { Network } from './Network/Network.jsx';
+import { SidebarOutline } from '../../../_general/icons/SidebarOutline.jsx';
+
+export const Sidebar = ({ isOpen, setIsOpen }) => {
+  const handleClick = () => setIsOpen(!isOpen);
 
-export const Sidebar = () => {
   return (
     <div className={cn.sidebar}>
       {/*<Network />*/}
-      <Navigation />
+      <Navigation isOpen={isOpen} />
+      <button onClick={handleClick} className={cn.button}>
```

Comment:
Is this button unique or we can reuse general component?

Reply:
this button is unique